traversal/BUILD

```diff
@@ -0,0 +1,14 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+load("@vaticle_dependencies//tool/checkstyle:rules.bzl", "checkstyle_test")
+
+checkstyle_test(
```

Comment:
This package doesn't exist anymore! YOu can delete this

Reply:
was probably a m merge/rebase misstep, no big deal

Reply:
Done